packages/crayons-core/src/components/nested-select/readme.md

```diff
@@ -0,0 +1,147 @@
+# fw-nested-select
+
+fw-nested-select allows to display nested dropdown choices
```

Comment:
Please add the react samples as well. Kindly check other components where `react` samples are also added

Reply:
Done.